views/building-hours/index.js

```diff
@@ -54,6 +75,13 @@ export default class BuildingHoursView extends React.Component {
         renderRow={this._renderRow.bind(this)}
         pageSize={4}
         initialListSize={6}
+        removeClippedSubviews={false}  // remove after https://github.com/facebook/react-native/issues/8607#issuecomment-241715202
```

Comment:
We will want to add this to every list view temporarily, perhaps? It has happened on more than one list view (Important Contacts for example)